libs.go

```diff
@@ -15,7 +15,9 @@ func Sign(message string) string {
 		return ""
 	}
 	mac := hmac.New(sha1.New, secretKey)
-	io.WriteString(mac, message)
+	if _, err := io.WriteString(mac, message); err != nil {
+		panic(err)
```

Comment:
L19 is not candidate for `panic`, please update to -
```go
ERROR.Println(err)
return ""
```

or you can change the `if` condition with `err == nil` have encode string inside and have `return ""` at the end of the method.